### PR TITLE
Consumable

### PR DESCRIPTION
**Pull request by TehanovAnton**
#9 I create Consumable and CarConsumable models, add associations. Car can have many consumables

### CODE REVIEW

**Comment by TehanovAnton** [comment]
#9 Add business models
I create  Car and CarConsumable models, add associations. Car can have many consumables

**Comment by dmokreckiy** [comment]
@TehanovAnton
 * No PR description
 * Reviewer not set
 * Assignee not set (yourself)

File: app/models/car_consumable_value.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class CarConsumableValue < ApplicationRecord
```
Comment (dmokreckiy) [inline app/models/car_consumable_value.rb:1]: Is it a join table? If yes - naming should be different

File: db/migrate/20211229160418_create_car_consumable_values.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CreateCarConsumableValues < ActiveRecord::Migration[6.1]
+  def change
+    create_table :car_consumable_values, id: false do |t|
+      t.integer :car_id, null: false
+      t.integer :consumable_id, null: false
+      t.integer :value, null: false
```
Comment (dmokreckiy) [inline db/migrate/20211229160418_create_car_consumable_values.rb]: OK, got it.

File: app/models/car_consumable_value.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  belongs_to :car, dependent: :destroy
+  belongs_to :consumable, dependent: :destroy
```
Comment (dmokreckiy) [inline app/models/car_consumable_value.rb:5]: validates presence of value? numerical?

**Review by dmokreckiy — CHANGES_REQUESTED** [review]
Add `values` validations and good to go.

**Review by yury-yaroshyk-itechart — APPROVED** [review]

**Pull request merged**